testing_and_setup/compass/ocean/global_ocean/CUSP12/spin_up/config_driver.xml

```diff
@@ -1,4 +1,4 @@
-<driver_script name="run_test.py">
+<driver_script name="run.py">
```

Comment:
This and similar lines in other `spin_up` driver scripts seem like they should be changed as above to be consistent with `init` driver scripts.  @mark-petersen, let me know if you disagree.

Reply:
Agreed. Easier to be consistent and only use run.py